frontend/simple/views/components/Transitions/MaskToModal/mixinTransitions.js

```diff
@@ -0,0 +1,167 @@
+    },
+    triggerLeave (el, complete) {
+      console.log('triggerLeave')
+      this.updateSpecsOf(el, 'trigger')
+
+      // Fade out the targetInner after the user interaction...
+      Velocity(el, { opacity: 0 }, { duration: this.config.fade, delay: this.config.fadeDelay, complete })
+    },
+
+    // -- Target animations
+    targetEnter (el, complete) {
+      console.log('targetEnter')
+      const targetInner = this.getTargetInnerWhen('enter')
+      this.updateSpecsOf(targetInner, 'target')
+
+      this.elementStartsInvisible(el)
+      this.elementStartsInvisible(targetInner)
+
+      // Fade in the Target after the Trigger has fade out...
+      Velocity(el, { opacity: 1 }, { duration: this.config.fade, delay: this.config.fade })
+      // And Fade in the targetInner only after the Masker has completed its transition animation
+      Velocity(targetInner, { opacity: 1 }, { duration: this.config.fade, delay: this.config.delay, complete })
+    },
+    transitionAfterEnter (el) {
+      console.log('transitionAfterEnter')
+      // Trigger/Target has opacity: 0 by default, so let's force to stay 1 after the animation finishes.
+      Velocity(el, { opacity: 1 }, { duration: 0 })
+    },
+    targetLeave (el, complete) {
+      console.log('targetLeave')
+      const targetInner = this.getTargetInnerWhen('leave')
+      this.updateSpecsOf(targetInner, 'target')
+
+      // Fade out the targetInner after the user interaction...
+      Velocity(targetInner, { opacity: 0 }, { duration: this.config.fadeBack / 2, delay: this.config.fadeBack / 2 })
+
+      // But only fades out complety the Target (that has the dark background)
+      // after Masker goes back to the initial position
+      Velocity(el, { opacity: 0 }, { duration: this.config.fade, delay: this.config.maskerTime, complete })
+    },
+    elementStartsInvisible (el) {
+      // Use only opacity to fadeIn/Out because it's faster than
+      // Velocity's 'fadeIn' feature (it doesn't use display)
+      Velocity(el, { opacity: 0 }, { duration: 0 })
+    },
+
+    // -- Masker animations
+    // when animates to Target
+    maskEnter (el, complete) {
+      console.log('maskLeave')
+
+      this.maskerTakesImmediatelyTheShapeOf(el, 'trigger', this.config.fade)
+      this.maskerAnimatesToTheShapeOf(el, 'target', complete)
+    },
+
+    // when animates back to a Trigger
+    maskLeave (el, complete) {
+      console.log('maskEnter')
+
+      this.maskerTakesImmediatelyTheShapeOf(el, 'target', this.config.fadeBack)
+      this.maskerAnimatesToTheShapeOf(el, 'trigger', complete)
+    },
+
+    // -- Animation utils
+    maskerTakesImmediatelyTheShapeOf (maskerEl, originalElement, duration) {
+      // Masker takes imediately the specs (dimensions and position)
+      // of the originalElement (Trigger or Target) size...
+      Velocity(maskerEl, { opacity: 0, ...this.getSpecsOf(originalElement) }, { duration: 0 })
```

Comment:
Here's another `duration: 0` where this can probably be replaced with non-Velocity code

Reply:
This one might be harder to replace. You don't have to if it's too hard. But my guess would be something like:

```js
Object.assign(maskerEl.style, { opacity: 0, ...this.getSpecsOf(originalElement) })
```

But that might not work.